src/renderer/shared/constants/testIds.ts

```diff
@@ -0,0 +1,26 @@
+export const TEST_IDS = {
+  ONBOARDING: {
+    VAULT_BUTTON: 'onboarding-vault-button',
+    WATCH_ONLY_BUTTON: 'onboarding-watch-only-button',
+    NOVA_WALLET_BUTTON: 'onboarding-nova-wallet-button',
+    WALLET_CONNECT_BUTTON: 'onboarding-wallet-connect-button',
+    LEDGER_BUTTON: 'onboarding-ledger-button',
+    WALLET_NAME_INPUT: 'onboarding-wallet-name-input',
+    WALLET_ADDRESS_INPUT: 'onboarding-wallet-address-input',
+  },
+  ASSETS: {
+    SETTINGS_WIDGET: 'assets-settings-widget',
+  },
+  MAIN: {
+    WALLET_BUTTON: 'main-wallet-button',
+    BACK_BUTTON: 'main-back-button',
+    CONTINUE_BUTTON: 'main-continue-button',
+    INFO_BUTTON: 'main-info-button',
+  },
+  TRANSFER: {
```

Comment:
Maybe just `operations`?
`Network fee` is a really common thing for any kind of operation
Thus I suggest to not go deeper that 1 level of nesting, or we'll end up with file similar to `en.json` which is hard to parse visually.

Reply:
https://github.com/novasamatech/nova-spektr/pull/2668/commits/d6a12e55d588fc3eb8b699293ed50b70a3df73e8